### DISCUSSION TITLE
All about QT-07S

### DISCUSSION THREAD

**Discussion by itepe** [Koenkk/zigbee2mqtt#18109] [Devices]
Hi everyone!
Today I am getting mentioned sensor. Seem to be solid and really waterproof.
Successfully added to to Home Assistant, but as **TS0601_soil** [https://www.zigbee2mqtt.io/devices/TS0601_soil.html](url) . On the website I see **TuYa QT-07S** [https://www.zigbee2mqtt.io/devices/QT-07S.html](url) as separate device. In both devices details I see some differences, in example **illuminance_calibration**.
OK, my question is: as is should identified my device? Is everything OK? I'm not Zigbee expert, so I need yours help.

Thanks in advance!

Kris

**Comment by edegraaff** [comment]
Hi, i also havethis device and not yet been able to pair it propperly. I do see a red led flashing quickly after pressing the button for 5 seconds but zigbee2mqtt did not pick it up. Tested other devices and here it worked, so i get the feeling the type of zigbee receiver is also important. Since it is not yet working at my site, what mqtt device do you use ?

**Reply by lbarjak** [reply]
- Take out the batteries
- wait five minutes
- put it back, do not press the button
- it will connect itself to zigbee2mqtt (remember, Permit join is required)
I have a SONOFF Zigbee 3.0 USB Dongle-E

**Reply by MilesTEG1** [reply]
> * Take out the batteries
> * wait five minutes
> * put it back, do not press the button
> * it will connect itself to zigbee2mqtt (remember, Permit join is required)
>   I have a SONOFF Zigbee 3.0 USB Dongle-E

I try to do this, without success to have QT-07S instead of TS0601_soil.
edit: i use Z2M update 1.39.1-1

**Comment by chfink0** [comment]
Hi, did you happen to be able to pair this device? I have the same issue with two of these sensors, red led is flashing but not connecting...Any clue?

**Comment by DeaneWheresItAt** [comment]
I'm having the same issue. It connects but says Not supported in MQQT.

**Comment by elektrikal80** [comment]
The issue who i have is that zigbee2mqtt te soil sensor( QT-07S) see but the sensor is not working in the log i see,                                  "Device '0xa4c13839a9068c3e' with Zigbee model 'TS0601' and manufacturer name '_TZE204_myd45weu' is NOT supported,"
what can i do to make it work ?

**Reply by marcobeltempo** [reply]
Support for the `_TZE204_myd45weu` fingerprint was added in https://github.com/Koenkk/zigbee-herdsman-converters/pull/7593.
I can confirm that the `QT-07S` sensor works on the dev branch. 

[Switch to the dev branch instructions](https://www.zigbee2mqtt.io/advanced/more/switch-to-dev-branch.html)

**Comment by wlan4you** [comment]
Hello,<

i have buy 2 from these QT-07S Devices.
https://www.zigbee2mqtt.io/devices/QT-07S.html

{
  "linkquality": 255
}

That is the only information over MQTT what i become to send.

Any ideas for me ?

**Comment by MilesTEG1** [comment]
Hello here,
with the Z2M update 1.39.1-1, it should be recongnized as QT-07S device, but for me it's not the case, t's still recognized as [TS0601_soil](https://www.zigbee2mqtt.io/devices/TS0601_soil.html#tuya-ts0601_soil).
I force removed it, and pair it again, but still:
![CleanShot 2024-08-02 at 07 29 39@2x](https://github.com/user-attachments/assets/eafa6a37-b7e6-430f-bbe8-f857c76fd368)
Is there anything I can do to try to get the QT-07S?

**Reply by wlan4you** [reply]
Thanks for the nformation.
I have send it back to china.
I would normaly two from this one for my wifes Glasgardenhouse with Tomatos and Salat inside.

And your Model is working fine with Z2M?
Have you a link where you have buyed them?

Thanks
